linchpin/api/__init__.py

```diff
@@ -188,7 +188,7 @@ def run_playbook(self, pinfile, targets='all', playbook='up'):
         if self.ctx.cfgs.get('ansible'):
             ansible_console = ast.literal_eval(self.ctx.cfgs['ansible'].get('console', 'False'))
 
-        self.ctx.evars['creds_path'] = self.ctx.creds_path
+        self.set_evar('creds_path', self.ctx.creds_path)
```

Comment:
This does not help. self.ctx.creds_path is being set into the same place. Why is this necessary?